azavea/iow-boundary-tool
# Create Django Backend

## Overview

This work has been to create the django backend and creates an example endpoint to demonstrate the functionality

Connects #6 

### Demo

```
$ http http://localhost:8181/api/example/

HTTP/1.1 200 OK
Connection: close
Content-Length: 20
Content-Type: text/plain; charset=UTF-8
Date: Wed, 08 Jun 2022 21:03:14 GMT
Referrer-Policy: same-origin
Server: gunicorn
X-Content-Type-Options: nosniff
X-Frame-Options: DENY

Example API response
```

### Notes



## Testing Instructions

 * check out the branch
 * run `./scripts/update`
 * run `./scripts/server`
 * test the API endpoint as seen above using httpie, or some other API testing utility
